### PR TITLE
Added date search functionality

### PR DESCRIPTION
**Pull request by dyampols**
- Now start date and end date (seasonal opening) are taken into account when recommending attractions 
- In database attractions that are open year round are represented as opening date = 2020-01-01 and end date = 2022-12-31 this is because it is assumed that past 2022 information is not available at the moment 
- this means that if a user selects dates past 2022 no attractions will be recommended 
- Ideally a user would not be able to select those days in the front end, it depends on @syasrebii findings on blocking those dates
- if it is possible to block,, they should be blocked from selection

### CODE REVIEW

**Comment by emilynairnn** [comment]
Looks good to me!

File: travvy-server/src/models/Attractions.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ module.exports = (sequelize, DataTypes) => {
     email_address: DataTypes.TEXT,
     phone_number: DataTypes.TEXT,
     street_address: DataTypes.TEXT,
-    opening_date: DataTypes.TEXT,
-    closing_date: DataTypes.TEXT,
+    opening_date: DataTypes.DATE,
+    closing_date: DataTypes.DATE,
```
Comment (lkdfount) [inline travvy-server/src/models/Attractions.js:17]: This makes way more sense than text thanks for changing it

**Review by lkdfount — APPROVED** [review]
Looks great thanks

**Comment by syasrebii** [comment]
Perfect, I'm working on making sure past dates cannot be selected from our calendar on the frontend. I could potentially use December 2022 as the "max" for the dates, since that's our upper bound in the database, will continue to play around. Thanks, Daria!

**Pull request merged**